time-series/time-series-api/src/main/java/com/powsybl/timeseries/CalculatedTimeSeriesDslLoader.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.timeseries;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.timeseries.ast.NodeCalc;
+
+import java.util.Map;
+import java.util.ServiceLoader;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public interface CalculatedTimeSeriesDslLoader {
+
+    Map<String, NodeCalc> load(String script, ReadOnlyTimeSeriesStore store);
+
+    static CalculatedTimeSeriesDslLoader find() {
```

Comment:
Should we have have a "name" identifier for dsl loaders? Or we take one randomly? Please note that it may be done in another PR.

Reply:
As there is no plan for adding a second DSL language and worst no plan for dealing with 2 different DSL scripts engine at the same time in the classpath I would stay with a simple impl that suppose there is only one in the classpath.
So you agree with this proposition, my code is not correct as we should also check there is only one and I will add a check to ensure unicity.